lib/KCard/index.vue

```diff
@@ -34,7 +34,7 @@
           as progressive loading experience.
         -->
         <span
-          v-if="$slots.thumbnailPlaceholder"
+          v-if="$slots.thumbnailPlaceholder && !thumbnailSrc"
```

Comment:
Even though the major function of the placeholder background and this slot's content is to serve as placeholder when the thumbnail is not available at all, it also has the additional function to show at least something while the image is loading over it. I've tried to explain with the comment above on line 33.

Adding this condition would break this additional feature. I assume there was a situation where not having the condition caused some unexpected problems - could you elaborate what was the motivation for this change? Perhaps we can think of a solution that would still allow us to keep the progressive loading experience.

Reply:
The motivation around this was to prevent the placeholder from showing when the thumbnailSrc is provided.

Reply:
Actually - on a second thought . we actually don't need both. This will do` v-if="$slots.thumbnailPlaceholder"`

Reply:
> was to prevent the placeholder from showing when the thumbnailSrc is provided

Ah okay, well we'd want to show it even in this case, if the slot is provided, for the reasons I mentioned above.

So if

> This will do v-if="$slots.thumbnailPlaceholder"

is sufficient, let's just keep it that way.

Reply:
> was to prevent the placeholder from showing when the thumbnailSrc is provided

Ah okay, well we'd want to show it even in this case, if the slot is provided, for the reasons I mentioned above -  to achieve something being displayed below the image when it's still loading, for example on slower networks

So if

> This will do `v-if="$slots.thumbnailPlaceholder"`

is sufficient, let's just keep it that way.